web/webhook_test.go

```diff
@@ -205,8 +205,9 @@ func TestIncomingWebhook(t *testing.T) {
 
 		hook, err := th.App.CreateIncomingWebhookForChannel(th.BasicUser.Id, th.BasicChannel, &model.IncomingWebhook{ChannelId: th.BasicChannel.Id, ChannelLocked: true})
 		require.Nil(t, err)
+		require.NotNil(t, hook)
 
-		url := ApiClient.Url + "/hooks/" + hook.Id
+		url = ApiClient.Url + "/hooks/" + hook.Id
```

Comment:
I'm almost sure reusing this variable is not only confusing, but also dangerous. If we run the tests in parallel, it can fail because it can modify the url for all the `t.Run`. I would rename the variable to something like `apiHookUrl` or whatever, but not reuse it.

Reply:
Fixed

Reply:
The `url` package is perhaps the prototypical argument to ignore shadowing, and we see the stdlib doing exactly that. I'm somewhat less concerned about shadowing a package, since it seems harder to "get wrong".

Anyway, I like `apiHookUrl` as @jespino suggests above.

Reply:
Just to make this clear: We were shadowing a variable `url`, not the package name.